modules/dev-tools/deprecation.php

```diff
@@ -159,15 +159,28 @@ public function compare_version( $version1, $version2 ) {
 		return false;
 	}
 
-	public function deprecated_function( $function, $version, $replacement = '', $base_version = null ) {
+	/**
+	 * Check Deprecation
+	 *
+	 * Checks whether the given entity is valid. If valid, this method checks whether the deprecation
+	 * should be soft (browser console notice) or hard (use WordPress' native deprecation methods).
+	 *
+	 * @param $type - Deprecation type (function|hook|argument... etc)
+	 * @param $entity - The Deprecated entity (the function/hook itself)
+	 * @param $version
+	 * @param $replacement
+	 * @param $base_version
+	 * @return bool|void
```

Comment:
1. Add the type of the parameter, mention if it's optional, and add the default value
2. Add an empty line before `@return`

The same goes for the other methods. See an example in the comment below.

Reply:
Done.